worldengine/cli/main.py

```diff
@@ -105,6 +107,8 @@ def generate_plates(seed, world_name, output_dir, width, height,
 
     # Generate images
     filename = '%s/plates_%s.png' % (output_dir, world_name)
+    # TODO calculate appropriate sea_level
+    sea_level = 1.0
```

Comment:
This is more stuff that has been changed recently. I hope we find a way to clearly separate the actual new code from regressions or a lot of things might break.